js.js

```diff
@@ -0,0 +1,40 @@
+const books = [];
+const addBtn = document.getElementById('add-new-book');
+const booksoutput = document.querySelector('#booksoutput');
+
+function outputbooklist() {
+  const booklist = JSON.parse(localStorage.getItem('books'));
+  booksoutput.innerHTML = '';
+  booklist.forEach((book, index) => {
+    booksoutput.innerHTML += `<div class = "addedbook"><p>${book.title}</p><p>${book.author}</p><button onclick='removefromlist(${index})'>Remove</button></div`;
+  });
+}
+
+function removefromlist(index) {
+  const booklist = JSON.parse(localStorage.getItem('books'));
+  booklist.splice(index, 1);
+  books.splice(index, 1);
```

Comment:
- [ ] OPTIONAL
It will be a good practice to favor the use of array methods such as `slice` over `spilce`, which doesn't mutate the array.